docs/user-guide/index.md

```diff
@@ -1,17 +1,14 @@
-# Getting Started
+# Browsertrix User Guide
 
-## Signup
+Welcome to the Browsertrix user guide! While our aim is to create intuitive interfaces, sometimes the complexities of web archiving require a little more explanation. If there's something that you found especially confusing or frustrating [please get in touch](docs-feedback@webrecorder.net)!
```

Comment:
I feel like this should be at the end rather than the beginning. At this point, the reader hasn't even seen any of the guide yet to be able to decide if they need to reach out.
Perhaps it could be a footer on all the docs pages?